requirements.txt

```diff
@@ -1,7 +1,5 @@
 # In-house dependencies
-leaf_common==1.2.21
-leaf_server_common==0.1.18
-neuro_san==0.5.20
+neuro_san==0.5.22
```

Comment:
Bumped neuro-san to 0.5.22 to include the version of `StreamingInputProcessor` that manages sly_data